tests/gracetabnote_tests.ts

```diff
@@ -3,39 +3,34 @@
 //
 // GraceTabNote Tests
 
-/* eslint-disable */
-// @ts-nocheck
-
 import { VexFlowTests, TestOptions } from './vexflow_test_helpers';
 import { ContextBuilder } from 'renderer';
 import { Flow } from 'flow';
+import { Formatter } from 'formatter';
 import { GraceNoteGroup } from 'gracenotegroup';
+import { GraceTabNote } from 'gracetabnote';
+import { TabNote, TabNoteStruct } from 'tabnote';
+import { TabStave } from 'tabstave';
+import { RenderContext } from 'types/common';
+import { Voice } from 'voice';
 
 const GraceTabNoteTests = {
   Start(): void {
     QUnit.module('Grace Tab Notes');
-    VexFlowTests.runTests('Grace Tab Note Simple', this.simple);
-    VexFlowTests.runTests('Grace Tab Note Slurred', this.slurred);
-  },
-
-  setupContext(options, x): void {
-    const ctx = options.contextBuilder(options.elementId, 350, 140);
-    const stave = new VF.TabStave(10, 10, x || 350).addTabGlyph().setContext(ctx).draw();
-
-    return { context: ctx, stave: stave };
+    // TODO: Rename tests below since it is redundant with the module name.
```

Comment:
+1

Reply:
Re: renaming test cases.

Since I've spent a lot of time in `flow.html` recently, I've noticed that many test cases have redundant naming since we use the Module Name > Test Case Name > Renderer + Font Name.

For example, "Accidental › Accidental Padding › Canvas + Bravura" or "Grace Tab Notes › Grace Tab Note Slurred › SVG + Bravura".

We will need to explore this in a separate PR, since renaming test cases is not trivial.

It **breaks the visual diffing** between versions, since it is done by file name. I haven't thought of a great way to preserve diffing between versions other than having a lookup table to translate test names between versions.

Reply:
Outside of manually adding stable identifiers for each test (or maybe generating them from file + function name), there isn't a better way.

This was something I never anticipated -- we can discuss for a separate PR.